spanner/tests/system.py

```diff
@@ -75,9 +85,6 @@ def setUpModule():
     Config.INSTANCE_CONFIG = configs[0]
     config_name = configs[0].name
 
-    def _list_instances():
-        return list(Config.CLIENT.list_instances())
-
     instances = retry(_list_instances)()
```

Comment:
I have an allergy to dynamically defined functions and classes. If nothing else, they make the flow of code harder to reason about (for both scope and indentation reasons).